src/Equinox.MessageDb/MessageDbClient.fs

```diff
@@ -0,0 +1,121 @@
+namespace Equinox.MessageDb
```

Comment:
this, and anythng else possible should prob be in a .Core namespace to constrain autocompletion